hugo/layouts/partials/tabs.html

```diff
@@ -20,7 +20,14 @@
                 data-tab-item="{{ .name | urlize }}"
                 data-tab-group="{{ $groupId }}"
                 class="tabs-nav__tab{{ if ne .type "default" }} tabs-nav__tab--{{ .type}}{{ end}}{{ cond (eq $idx 0) " is-active" ""}}"
-            >{{ .name }}</button>
+            >
+                {{- if and (eq .type "default") (eq .fileIcon true) -}}
+                    <div class="tabs-nav__copy" data-tab-copy data-tab-copy-value="{{ .name }}">
```

Comment:
a clickable item should be a button (accesiblity)., a div is not accessible by keyboard .

Reply:
> updated to button for line 50-53 
> for line 25-29 a nested button inside another button is not valid or recommented. This causes rendingering and behavior issues. I replaced it with a span element.